src/pages/ProductsPage.tsx

```diff
@@ -46,68 +81,126 @@ const ProductsPage = () => {
 								<CategoryModel openModel={openModel} />
```

Comment:
You can improve this by automatically closing the model when the use click on one category, i think it may be a better experience. Cuase you can see how this look in ipad and mobile also. Apart from that, everything is good.
<img width="1440" alt="Screenshot 2024-07-07 at 09 55 54" src="https://github.com/atlp-rwanda/hackers-ec-Fe/assets/151399071/17beaa37-3bbd-482a-a371-03f60728bcd5">
<img width="1440" alt="Screenshot 2024-07-07 at 09 56 08" src="https://github.com/atlp-rwanda/hackers-ec-Fe/assets/151399071/bf244742-a18a-4207-a650-f9e9b6cbf3a7">